tag.js

```diff
@@ -27,18 +27,17 @@ const opts = nomnom
     })
     .parse();
 
-return index.tagTemplate({
+index.tagTemplate({
```

Comment:
Why are we getting rid of the `return`?

Reply:
`return` isn't needed, and seems out of place since it doesn't reside in a function body. I've decided to keep it though, just for the sake of consistency.